### PR TITLE
feat(cctp): bridge ui design updates + input/output token support

### PR DESCRIPTION
**Pull request by dohaki**
Attempt to update production with cumulated changes from:
- #1004 
- #1012

### CODE REVIEW

File: api/_utils.ts [hunk 1]
```diff
@@ -397,13 +397,13 @@ export const getRouteDetails = (
 
 const _getTokenByAddress = (tokenAddress: string, chainId?: number) => {
   const [, token] =
-    Object.entries(TOKEN_SYMBOLS_MAP).find(([_symbol, { addresses }]) =>
-      chainId
+    Object.entries(TOKEN_SYMBOLS_MAP).find(([_key, { addresses }]) => {
+      return chainId
```
Comment (james-a-morris) [inline api/_utils.ts:401]: Do we need the return?

File: src/hooks/useBridgeFees.ts [hunk 2]
```diff
@@ -18,34 +17,38 @@ export function useBridgeFees(
...
+    ({ queryKey }) => {
+      const [
+        ,
+        tokenSymbolToQuery,
+        amountToQuery,
+        fromChainIdToQuery,
+        toChainIdToQuery,
+      ] = queryKey;
+
+      if (!toChainIdToQuery || !fromChainIdToQuery || !tokenSymbolToQuery) {
```
Comment (james-a-morris) [inline src/hooks/useBridgeFees.ts:37]: Thoughts on switching this to use `isDefined( )`?

File: src/views/Bridge/components/RecipientRow.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  onClickChangeToAddress: VoidHandler;
+};
+
+export function RecipientRow(props: Props) {
+  const { data } = useEnsQuery(props.recipient.address);
+
+  return (
+    <Wrapper>
+      <Text color="grey-400">Address:</Text>
+      {data?.avatar && <img src={data.avatar} alt="avatar" />}
```
Comment (james-a-morris) [inline src/views/Bridge/components/RecipientRow.tsx:22]: Could we use a next Image tag here?

**Review by james-a-morris — APPROVED** [review]
a few nits - otherwise LGTM

File: src/hooks/useBridgeFees.ts (hunk 2 above)
Comment (dohaki) [reply src/hooks/useBridgeFees.ts:37]: Why do you prefer `isDefined` here`?

File: src/views/Bridge/components/RecipientRow.tsx (hunk 3 above)
Comment (dohaki) [reply src/views/Bridge/components/RecipientRow.tsx:22]: We never used `next/image` in the dApp so far... If we introduce it, I would prefer to do so in a separate PR

File: api/_utils.ts (hunk 1 above)
Comment (dohaki) [reply api/_utils.ts:401]: Reverted

**Pull request merged**